tests/testthat/test-template.R

```diff
@@ -46,6 +46,27 @@ test_that("UTF-8 characters in templates", {
   expect_identical(charToRaw(text), as.raw(c(0xc3, 0xbf)))
 })
 
+test_that("UTF-8 characters in template head but not body", {
+  # On Windows, a string with "中文" will automatically be marked as UTF-8.
+  ui <- tagList(
+    tags$head(tags$script("alert('中文')")),
```

Comment:
My Windows VM definitely doesn't have its locale set to Chinese. But it was using a recent version of Windows; I agree that it's possible that older versions of windows might not handle this the same way. I'd prefer using `rawToChar()` instead (like in the previous test) instead of skipping the test.

It makes sense in this case to explicitly mark the string as UTF-8 on all platforms, but for the next test (with `á`) I'm not sure what the right thing to do is. On Unixes, the string would be automatically marked as UTF-8, but on Windows, it gets marked as latin1 (at least it did on my VM).